src/app/shared/models/fcmNotificationDto.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * App Server
+ * RADAR App Server Api Documentation
+ *
+ * OpenAPI spec version: 0.0.1-SNAPSHOT
+ *
+ *
+ * NOTE: This class is auto generated by the swagger code generator program.
+ * https://github.com/swagger-api/swagger-codegen.git
+ * Do not edit the class manually.
+ */
+
+export interface FcmNotificationDto {
+  appPackage?: string
+  body?: string
+  delivered?: boolean
+  fcmMessageId?: string
+  id?: number
+  scheduledTime?: Date
+  sourceId?: string
+  sourceType?: string
+  title?: string
+  ttlSeconds?: number
+  type?: string
```

Comment:
body, title, type, sourceId,  ttlSeconds, appPackage, scheduledTime cannot be null. fcmMessageId can be omitted as it is calculated dynamically on the appserver using Notification entity's hashcode

Reply:
I know this is auto-generated by codegen, but i thought we are using the dynamic client (so maybe ok to edit this manually??)

Reply:
Ah yes we're using the dynamic client. I just kept the autogenerated models so I didn't have to manually create it. I'll make the changes.